js_modules/dagster-ui/packages/ui-core/src/instigation/TickDetailsDialog.tsx

```diff
@@ -27,70 +71,197 @@ export const TickDetailsDialog: React.FC<{
+  const intent = React.useMemo(() => {
+    switch (tick?.status) {
+      case InstigationTickStatus.FAILURE:
+        return 'danger';
+      case InstigationTickStatus.STARTED:
+        return 'primary';
+      case InstigationTickStatus.SUCCESS:
+        return 'success';
+    }
+    return undefined;
+  }, [tick]);
+
+  const isAssetDaemonTick = 'requestedAssetMaterializationCount' in tick;
+
+  return (
+    <>
+      {showPythonError ? (
+        <Dialog
+          title="Error"
+          isOpen={!!showPythonError}
+          onClose={() => {
+            setShowPythonError(null);
+          }}
+          style={{width: '80vw'}}
+        >
+          <DialogBody>
+            <PythonErrorInfo error={showPythonError} />
+          </DialogBody>
+          <DialogFooter topBorder>
+            <Button
+              intent="primary"
+              onClick={() => {
+                setShowPythonError(null);
+              }}
+            >
+              Close
+            </Button>
+          </DialogFooter>
+        </Dialog>
+      ) : null}
+      <div style={{display: 'grid', gridTemplateColumns: 'repeat(3, minmax(0, 1fr))', gap: 12}}>
+        <Box flex={{direction: 'column', gap: 4}}>
+          <Subtitle2>Status</Subtitle2>
+          <Box flex={{direction: 'row', gap: 4, alignItems: 'center'}}>
+            <Tag intent={intent}>
+              {tick.status === InstigationTickStatus.STARTED ? (
+                'Evaluating…'
+              ) : (
+                <>
+                  {(isAssetDaemonTick
+                    ? tick.requestedAssetMaterializationCount
+                    : tick.runIds.length) ?? 0}{' '}
+                  requested
+                </>
+              )}
+            </Tag>
+            {tick.error ? (
+              <ButtonLink
+                onClick={() => {
+                  setShowPythonError(tick.error!);
+                }}
+              >
+                View error
+              </ButtonLink>
+            ) : null}
+          </Box>
+        </Box>
+        <Box flex={{direction: 'column', gap: 4}}>
+          <Subtitle2>Timestamp</Subtitle2>
+          <div>{tick ? <Timestamp timestamp={{unix: tick.timestamp}} /> : '–'}</div>
+        </Box>
+        <Box flex={{direction: 'column', gap: 4}}>
+          <Subtitle2>Duration</Subtitle2>
+          <div>
+            {tick?.endTimestamp
+              ? formatElapsedTime(tick.endTimestamp * 1000 - tick.timestamp * 1000)
+              : '–'}
+          </div>
+        </Box>
+      </div>
+    </>
+  );
+}
+
+function PartitionsTable({partitions}: {partitions: DynamicPartitionsRequestResult[]}) {
+  return (
+    <Table>
+      <thead>
+        <tr>
+          <th>Asset</th>
+          <th>Partition</th>
+        </tr>
+      </thead>
+      <tbody>
+        {partitions.flatMap(
+          (partition) =>
+            partition.partitionKeys?.map((key) => (
+              <tr key={key}>
+                <td>{partition.partitionsDefName}</td>
```

Comment:
I'm not familiar with this GraphQL schema but just want to sanity check that "Asset" is the right name for this column?

Reply:
Oh oops, yeah it should be partition definition